Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Batch.java

```diff
@@ -102,7 +102,7 @@ public Batch(String title, Collection<? extends Process> processes) {
     }
 
     /**
-     * The function getTitle() returns the batch title. Using titles for batches
+     * Returns the batch title. Using titles for batches
```

Comment:
I see here inconsistency - here Returns, few line lower Return.

Reply:
The `@return` is an instruction to the Javadoc generator. It translates to „Returns:“

![Anmerkung 2019-07-12 121221](https://user-images.githubusercontent.com/3040657/61120966-65219b00-a49e-11e9-90a1-07db59751eb4.png)